pom.xml

```diff
@@ -202,6 +202,7 @@
     <datanucleus-core.version>4.1.17</datanucleus-core.version>
     <guava.version>14.0.1</guava.version>
     <janino.version>3.1.9</janino.version>
+    <jaxb-api.version>2.2.11</jaxb-api.version>
```

Comment:
You can spin off this together with https://github.com/apache/spark/pull/43736/files#r1395282370